neurom/features/neuritefunc.py

```diff
@@ -533,7 +537,13 @@ def neurite_volume_density(neurites, neurite_type=NeuriteType.all):
     """
     def vol_density(neurite):
         """Volume density of a single neurite."""
-        return neurite.volume / convex_hull(neurite).volume
+        try:
+            volume = convex_hull(neurite).volume
+        except scipy.spatial.qhull.QhullError as e:
+            L.warning(e)
+            return np.nan
```

Comment:
I'm afraid I don't have a very good enough reason except that otherwise I can't build the dataframe made of all the features.
I have improved the error message though.